.vscode/launch.json

```diff
@@ -0,0 +1,53 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "jlab backend",
+            "type": "python",
+            "request": "launch",
+
+            "args": [
+                "--dev-mode",
+                "--notebook-dir=${env:HOME}",
+                "--no-browser",
+                "--NotebookApp.token=''",
+                "--port=9999"
+            ],
+            "module": "jupyterlab.labapp",
+
+            // can potentially run frontend using serverReadyAction instead
+            // of using compound job, currently not configurable enough
+            //
+            // "serverReadyAction": {
+            //     "pattern": "http://localhost:(\\S+)",
+            //     "uriFormat": "http://localhost:%s",
+            //     "action": "debugWithChrome",
+            //     "webRoot": "${workspaceFolder}"
+            // },
+        },
+        {
+            "name": "jlab frontend",
+            "type": "chrome",
+            "request": "launch",
```

Comment:
Maybe there could also be an `attach` configuration, to attach to an already running Chrome?

Chrome / Chromium would have to be started with the following (or equivalent):

```
/usr/bin/chromium-browser -remote-debugging-port=9222
```

Reply:
What's the use case for attach vs launch? I'm having trouble seeing any time I'd prefer to attach, especially since it requires that I launch chrome with a special invocation in the first place.

Reply:
I was wondering how it would work with multiple Chrome profiles that have different browser extensions installed.

Just tested and it looks like the new tab is opened in the first Chrome window, which is good.